src/components/common/CodeReview.css

```diff
@@ -30,11 +6,3 @@
 .style.token.string {
   background-color: aliceblue !important;
 }
```

Comment:
This is used to overwrite the background of the code area. 
`.code-line` is a class defined by us, the others are overriden to ensure that everything has the same background. Equal signs were having a different background. I think this could be achieved also by using a theme provided by Prismjs more cleanly (this is a bit hacky).

Reply:
OK